bootstrap/main.go

```diff
@@ -56,6 +56,8 @@ func mainImpl() {
 		"weave_cloud_hostname": opts.WCHostname,
 	})
 
+	// Pass flag to always skip checking server's certificate for validity.
+	otherArgs = append(otherArgs, "--insecure-skip-tls-verify=true")
```

Comment:
The comment seems kinda redundant. IF we are putting a comment here, it should explain _why_ we are skipping the cert validation.

Reply:
@rade Valid point, fixed. PTALA, thanks!